Program.cs

```diff
@@ -247,11 +247,21 @@ private void SyncMenuItemStates(StatusResult status)
                 copyOCLoginForDeveloperMenu.Enabled = false;
                 copyOCLoginForKubeadminMenu.Enabled = false;
             }
+            else if (!string.IsNullOrEmpty(status.Error) && status.Error.Contains("does not exist"))

```

Comment:
I would suggest to over time create a nicer switch structure that calls individual clearly named functions, as now it is hard to understand what is set for the menu items and what is different.